fhireval/test_suite/crud/test_condition.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+import pdb
+
+from fhir_walk.model import unwrap_bundle
+from fhireval.test_suite.crud import prep_server
+import urllib.parse
+
+test_id = f"{'2.5.08':10} - CRUD Condition"
```

Comment:
In `fhireval/test_suite/crud/__init__.py` the test set ID seems to be redacted to `"2.2"`?